packages/matchbox/src/components/ThemeProvider/theme.js

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable max-lines */
+import tokens from '@sparkpost/design-tokens';
```

Comment:
Is there any way we could have exported constants to use instead of having to depend on all of these string props? Feels like it's pretty susceptible to typos and we'd get free IDE integration for autocomplete. Haven't messed with styled components before, so I'm not sure how much of a pain it would be.